src/components/forms/file-dropper/file-list-item.tsx

```diff
@@ -104,7 +104,7 @@ export const FileListItem = ({
             ))}
           </FileCategoryDropdown>
         )}
-        <RemoveButton onClick={() => onRemove(index)}>
+        <RemoveButton type="button" onClick={() => onRemove(index)}>
```

Comment:
I noticed that when you click remove, the form is sometimes submitted by mistake. This shouldn't happen anymore.